cluster/addons/fluentd-elasticsearch/fluentd-es-image/build.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+# Copyright 2015 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Install prerequisites.
+apt-get update
+
+apt-get install -y -q --no-install-recommends \
+  curl ca-certificates make g++ sudo bash
+
+# Install Fluentd.
+/usr/bin/curl -sSL https://toolbelt.treasuredata.com/sh/install-ubuntu-xenial-td-agent2.sh | sh
+
+# Change the default user and group to root.
+# Needed to allow access to /var/log/docker/... files.
+sed -i -e "s/USER=td-agent/USER=root/" -e "s/GROUP=td-agent/GROUP=root/" /etc/init.d/td-agent
+
+# Install the Elasticsearch Fluentd plug-in.
+td-agent-gem install fluent-plugin-kubernetes_metadata_filter fluent-plugin-elasticsearch
```

Comment:
Should the versions be specified for compatibility? Subsequent builds of this image could install different versions if more recent versions have been released?